src/stats.rs.in

```diff
@@ -0,0 +1,48 @@
+#[derive(Debug, PartialEq, Eq, Deserialize, Clone)]
+pub struct Period {
+    pub day: String,
```

Comment:
It can have `NaiveDate` type from `chrono`. We will need to turn on `serde` feature for the `chrono` crate in `Cargo.toml` to get `Deserialize` implementation for `NaiveDate`, but it will make the structure usage more convenient.